examples/common/python/connectors/direct/direct_json_rpc_api_adaptor_factory.py

```diff
@@ -17,13 +17,13 @@
 import errno
```

Comment:
Since changes are better to split the changes to 2 PRs.
1 moving/fixing connector module compilation issues.
2 GUI client for heart disease eval workload

Reply:
Done.